src/Features/Core/Portable/EmbeddedLanguages/StackFrame/StackFrameCompilationUnit.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.EmbeddedLanguages.Common;
+
+namespace Microsoft.CodeAnalysis.EmbeddedLanguages.StackFrame
+{
+    internal class StackFrameCompilationUnit
+    {
+        public readonly EmbeddedSyntaxTrivia<StackFrameKind>? AtTrivia;
+        public readonly StackFrameMethodDeclarationNode MethodDeclaration;
+        public readonly EmbeddedSyntaxTrivia<StackFrameKind>? InTrivia;
+        public readonly StackFrameFileInformationNode? FileInformationExpression;
+        public readonly EmbeddedSyntaxTrivia<StackFrameKind>? TrailingTrivia;
```

Comment:
generally we store trailling trivia as the leading trivia of an EOF token... but that may be overkill here.

Reply:
I like this idea. Going to try to add this in next iteration.